src/substrait/textplan/converter/InitialPlanProtoVisitor.cpp

```diff
@@ -19,6 +19,7 @@ namespace io::substrait::textplan {
 
 namespace {
 
+const std::string kIntermediateNodeName = "intermediate";
```

Comment:
Let's pick one format for consistency.  Either `= ""` or `{""}`

Reply:
Done.